### DISCUSSION TITLE
Bank-specific Options

### DISCUSSION THREAD

**Discussion by jlauwers** [firefly-iii/firefly-iii#4686] [General]
Hello,

I just saw the message `These options are deprecated. They may be removed in a future version of the CSV importer. If you rely on any of these bank-specific options, discuss it on GitHub.` And I didn't see a discussion for it.

So here am I :)

I use the ING BE.

BTW you may want to show a warning for those who have the "skip form" feature activated.

**Comment by JC5** [comment]
I'll add the warning, thanks. For the ING BE specific code and the others, best I can do is separate scripts.

**Reply by Adrellias** [reply]
Hi Folks,

Hopefully nobody minds me adding to this :)

Also saw the message today :) Not sure if I have to create a separate issue, I use the ABN Amro functionality.
